velox/functions/prestosql/tests/SequenceTest.cpp

```diff
@@ -207,6 +207,18 @@ TEST_F(SequenceTest, dateInvalidIntervalDayStep) {
       expected);
 }
 
+TEST_F(SequenceTest, invalidDate) {
+  int64_t day = 86400000; // 24 * 60 * 60 * 1000
+  const auto startVector = makeFlatVector<int32_t>({1420919120, 1992}, DATE());
+  const auto stopVector = makeFlatVector<int32_t>({1476733096, 2000}, DATE());
+  auto stepVector =
+      makeFlatVector<int64_t>({1 * day, 0, 1}, INTERVAL_YEAR_MONTH());
```

Comment:
stepVector has different size than the startVector and stopVector? Probably vectors of size 1 is sufficient for this test.